typescript-test-samples/schema-and-contract-testing/tests/contract-testing.test.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import event_1_0_0 from '../events/event-1.0.0.json';
```

Comment:
Getting a warning from typescript here (though the tests still run)

`Cannot find module '../schemas/Json/CustomerCreated-v1.0.0.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.ts(2732)`

Could you investigate?

(this is happening for the other test file as well)

Reply:
fixed